dem2dged_geo.py

```diff
@@ -99,10 +136,17 @@ def main(args):
             dl.dp(" ")
             dl.dp("-"*70)
             dl.dp("Creating elevation raster %s" %(namnam))
-            cmdstr = """gdalwarp -t_srs EPSG:%s+3855 -te %s %s %s %s -dstnodata -32767 -tr %s %s -r cubic -co COMPRESS=LZW --config GTIFF_REPORT_COMPD_CS YES %s %s""" %(my_out_srs, minlon, minlat, maxlon, maxlat, lonres, latres, pargs.input_raster, namnam )
+            # EPSG:3855 gives the vertical reference of EGM2008. For a lot of files, gdalwarp doesn't work when this reference is given.
+#            cmdstr = """gdalwarp -t_srs EPSG:%s+3855 -te %s %s %s %s -dstnodata -32767 -tr %s %s -r cubic -co COMPRESS=LZW --config GTIFF_REPORT_COMPD_CS YES %s %s""" %(my_out_srs, minlon, minlat, maxlon, maxlat, lonres, latres, pargs.input_raster, namnam )
```

Comment:
Suggest to delete this line in order to clean up comments

Reply:
We agree. We will remove these lines

Reply:
It seems that the commented lines have already been removed.